masonry_testing/src/harness.rs

```diff
@@ -349,7 +350,6 @@ impl<W: Widget> TestHarness<W> {
         };
 
         // Set up the initial state, and clear invalidation flags.
-        harness.process_window_event(WindowEvent::Resize(window_size));
```

Comment:
As far as I understand, the initial size being zero was an intentional decision, as a sort of passive version of #788. For tests specifically, I think that defaulting to that behaviour is possibly actually a good thing.

Reply:
The initial window size in testing is controlled by `TestHarnessParams::window_size`, so if starting with zero is useful, those specific tests can set it accordingly. Indeed, that is exactly what the zero size tests added in #1232 do.

The removal of this specific event is because it's no longer needed and no longer matches actual app behavior. It would also always send the same value that the window was initialized with. (*There's probably value in emulating the behavior of the startup Winit resize event, but that would be a bit later, not at this moment in the lifecycle.*)

Reply:
The point is that this change reduces coverage of zero size handling for a large number of widgets.

Now maybe that's fine, and we probably should have such tests explicitly instead. But it is a regression in what's covered.